### PR TITLE
Results landing page

### PR DESCRIPTION
**Pull request by SeeYouThursday**
This PR includes changes to the results and confirming results pages to create a more responsive layout.

Know Issues: 
- Searching from initial results page, can lead to empty cards
- - Fix will be: write a function to clear those results if the search occurs in this instance
- Searching from final selection (confirm page) - does not clear selection summary, nutrition, or result card
- --fix will probably be a conditional statement to clear when search happens during this instance
- In summary on final selection page, some summaries refer to "directions below" 
- --fix will probably be getting rid of the summary OR providing a drop down/btn to toggle instructions

### CODE REVIEW

**Review by underwoodjo — APPROVED** [review]
Awesome work!
